doc/api/deprecations.md

```diff
@@ -3899,6 +3899,20 @@ of built-in modules. This was incomplete and matched the already deprecated
 `repl._builtinLibs` ([DEP0142][]) instead it's better to rely
 upon `require('node:module').builtinModules`.
 
+### DEP0192: `require('node:_tls_common')` and `require('node:_tls_wrap')`
+
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/57643
+    description: Runtime deprecation.
+-->
+
+Type: Runtime
+
+The `node:_tls_common` and `node:_tls_wrap` modules are deprecated as they should be considered
+an internal nodejs implementation rather than a public facing API, use `node:tls` instead.
```

Comment:
Just noting... it's an unfortunate truth that while these are deprecated we likely will never be able to actually remove them. See `require('sys')` ....

Reply:
oh I see.. that's pretty disappointing :smiling_face_with_tear: (although I understand the security implications...)

I like [your suggestion of throwing an error](https://github.com/nodejs/node/pull/26292#pullrequestreview-207483471) though, it'd be pretty nice if we could do that at some point :slightly_smiling_face:

Reply:
mh... or maybe the _ modules could be removed at some point since that npm doesn't support such modules anyways? :thinking:
 
https://github.com/nodejs/node/pull/26292#issuecomment-467320578
https://github.com/npm/validate-npm-package-name?tab=readme-ov-file#naming-rules

:thinking:

Reply:
npm may not but an attacker could still locally inject a module which would still be problematic.

Reply:
oh I see what you mean... like with a postinstall script or something that drops an _ module in the node_modules right? :thinking: 

oh too bad then :sweat: (but I am still hoping for the throwing idea! :grinning:)